src/common/notifications/notifications_dll.rc

```diff
@@ -0,0 +1,35 @@
+#include "resource.h"
+#include "../../../common/version.h"
+
+1 VERSIONINFO
+ FILEVERSION FILE_VERSION
+ PRODUCTVERSION PRODUCT_VERSION
+ FILEFLAGSMASK 0x3fL
+#ifdef _DEBUG
+ FILEFLAGS 0x1L
+#else
+ FILEFLAGS 0x0L
+#endif
+ FILEOS 0x40004L
+ FILETYPE 0x2L
+ FILESUBTYPE 0x0L
+BEGIN
+    BLOCK "StringFileInfo"
+    BEGIN
+        BLOCK "040904b0"
+        BEGIN
+            VALUE "CompanyName", COMPANY_NAME
+            VALUE "FileDescription", "Notifications PowerToy Module"
+            VALUE "FileVersion", FILE_VERSION_STRING
+            VALUE "InternalName", "Notifications PowerToy"
+            VALUE "LegalCopyright", COPYRIGHT_NOTE
+            VALUE "OriginalFilename", "notifications_powertoy.dll"
+            VALUE "ProductName", "Notifications PowerToy"
+            VALUE "ProductVersion", PRODUCT_VERSION_STRING
+        END
+    END
+    BLOCK "VarFileInfo"
+    BEGIN
+        VALUE "Translation", 0x409, 1200
```

Comment:
⚠ Warning: 1200 codepage here doesn't match any block specified in this file.
Similar comment to above would be nice too.